src/Nest/Cluster/TaskManagement/GetTask/GetTaskResponse.cs

```diff
@@ -9,5 +10,10 @@ public class GetTaskResponse : ResponseBase
 
 		[DataMember(Name = "task")]
 		public TaskInfo Task { get; internal set; }
+
+		[DataMember(Name = "response")]
+		internal LazyDocument Response { get; set; }
+
+		public TResponse GetResponse<TResponse>() where TResponse : class, IResponse => Response?.As<TResponse>();
```

Comment:
Add some documentation around what properties on here make sense.

Reply:
I've addressed this in 9add6db. Also fixed a bug where the response must be deserialized with the `RequestResponseSerializer`